auslib/db.py

```diff
@@ -341,13 +382,17 @@ def _prepareInsert(self, trans, changed_by, **columns):
         if self.onInsert:
             self.onInsert(self, "INSERT", changed_by, query, trans)
 
+        potential_signoffs = self.getPotentialSignoffs(None, columns, transaction=trans)
+        if potential_signoffs:
```

Comment:
Oh wait, I guess I understood this condition wrong. "No potential signoffs" means "sometimes a product (for instance) doesn't require any sign off", right? If that's the case, I initially thought there was a way to contact this layer without any signoff at all.